### DISCUSSION TITLE
PEM key for github app

### DISCUSSION THREAD

**Discussion by brunolnetto** [community/community#34960] [Apps, API and Webhooks]
Hi,

I am aware of the great effort the GitHub staff community applies to keep documentation up to date: I learn a lot only by reading its instructions. However, the route https://docs.github.com/developers have some misleading instructions rather because of an interface update or some GitHub daemon altered setup. In particular, the PEM key generation and verification are not precisely as on documentation.  I suggest you set a senior development team to review it based on dev-user-experience requirements.

https://docs.github.com/pt/developers/apps/building-github-apps/creating-a-github-app

**Comment by ettaboyle** [comment]
@brunolnetto Thank you for this feedback. Can you please be more specific about what you found to be **misleading instructions**? We can share this feedback with the documentation team directly to improve the developer resources.

Thank you! ❤️
